lib/livereload.js

```diff
@@ -112,7 +113,12 @@ function StartServer() {
             if (err) {
                 deferred.reject(err);
             }
+
             var server_url = bs.options.getIn(['urls', 'external']);
+            if (!server_url) { // test for null. does this exit immediately or does it execute deferred.resolve as well ?
+                deferred.reject('No external url available');
```

Comment:
Can you describe when this happens in the error mesage and what should the user do to get around this?